.github/workflows/run-td-build.yml

```diff
@@ -44,4 +44,4 @@ jobs:
           azure-devops-project-url: 'https://skype.visualstudio.com/SCC'
           azure-pipeline-name: 'Azure.communication-ui-library-touchdown-build'
           azure-devops-token: '${{ secrets.AZURE_TOUCHDOWN_BUILD_PIPELINE_SECRET }}'
-          azure-pipeline-variables: '{"commit": ${{ steps.branch.outputs.branch == env.MAIN_BRANCH }}, "github_token": "${{ secrets.GITHUB_TOKEN }}"}'
+          azure-pipeline-variables: '{"commit": ${{ steps.branch.outputs.branch != env.MAIN_BRANCH }}, "github_token": "${{ secrets.GITHUB_TOKEN }}"}'

```

Comment:
Small fix here. Should not commit changes after touchdown build run if current branch is `main`.